UnitedIncome/components
# [APP-4294] Making sure that the colors.js file matches what is in the colors.scss file

### Description
> Please provide a brief description of your changes below.

Modified the `colors.js` file to match what is in the `colors.scss` file. I noticed that the `graph-ocean` color was missing initially when investigating a client bug so I did a sweep of all colors and found that we were also missing the `table-hover` color so I inserted that one as well.

### Testing Instructions
> What should be done in order to test this pull request, or re-create its effects.

See the related [Client-side PR](https://github.com/UnitedIncome/client/pull/3501).

### Screenshots
> If this pull request includes a visual change please attach any applicable screenshots.

### Comments
> Any additional comments that will aid with the review process.

- [client PR](https://github.com/UnitedIncome/client/pull/3501)
- [JIRA Ticket](https://unitedincome.atlassian.net/browse/APP-4294)
